docs/src/content/docs/guides/recurring-subscription-incoming-amount.mdx

```diff
@@ -0,0 +1,428 @@
+
+This approach is particularly useful for subscription service scenarios where:
+
+- The service provider charges a fixed monthly fee
+- The recipient (service provider) must receive exactly the subscription price each month
+- The sender (customer) authorizes recurring payments at a set interval
+- The sender wants to avoid manually approving each monthly payment
+
+## Scenario
+
+Imagine a customer subscribing to a streaming service. They want to authorize monthly payments of exactly \$15 USD for 12 months, and the service provider must receive the full \$15 USD each month to maintain the subscription.
+
+For this guide, you'll assume the role of a developer building a subscription management platform. The guide explains how to set up a \$15 USD monthly subscription payment that recurs for 12 months, where the recipient receives exactly \$15 USD each billing period.
+
+**Example transaction details:**
+
+- **Recipient receives**: \$15.00 USD (exact amount each month)
+- **Payment frequency**: Monthly for 12 months
+- **Sender pays**: \$15.00 USD each month (plus any applicable fees from their wallet provider)
+
+The three parties involved in this scenario are:
+
+- **Developer**: you, the person building the subscription management platform
+- **Sender**: the person subscribing to and paying for the service
+- **Recipient**: the service provider receiving the subscription payments
+
+Remember, Open Payments doesn't execute payments or touch money in any way. It's used to issue payment instructions before any money movement occurs. An example of a payment instruction is, "authorize recurring payments of exactly \$15 to the service provider once a month for 12 months".
+
+## Endpoints
+
+- <Badge text="GET" variant="note" /> <LinkOut href='https://openpayments.dev/apis/wallet-address-server/operations/get-wallet-address/'>Get Wallet Address</LinkOut>
+- <Badge text="POST" variant="success" /> <LinkOut href='https://openpayments.dev/apis/auth-server/operations/post-request/'>Grant Request</LinkOut>
+- <Badge text="POST" variant="success" /> <LinkOut href='https://openpayments.dev/apis/resource-server/operations/create-incoming-payment/'>Create Incoming Payment</LinkOut>
+- <Badge text="POST" variant="success" /> <LinkOut href='https://openpayments.dev/apis/resource-server/operations/create-outgoing-payment/'>Create an Outgoing Payment</LinkOut>
+- <Badge text="POST" variant="success" /> <LinkOut href='https://openpayments.dev/apis/auth-server/operations/post-token/'>Rotate an Access Token</LinkOut>
+
+## Steps
+
+### 1. Get wallet address information
+
+When the sender initiates a subscription, your platform needs to get wallet address information for both the sender and the recipient.
```

Comment:
```suggestion
When the customer signs up for a subscription, your platform needs to get wallet address information for both the customer and the service provider.
```

Reply:
As an example. This also keeps you from needing to put "(service provider)" in parens like you have in the next paragraph.